### ISSUE TITLE
Intuition/ref for -fc_lr_mul

### ISSUE THREAD

**Issue by MauritsBleeker** [Andrew-Brown1/Smooth_AP#19]
Hi Andrew,

Thanks for sharing this nice piece of work! 
I have a question about the flag -fc_lr_mul. So basically what you do here is scaling up the lr for the last linear layer times n.

Can you share how/why you decided to do this? And do you have any ablation results on how much the influence of this scaling is on the final scores?

Thanks!

Maurits

**Comment by Andrew-Brown1** [comment]
Hey Maurits,

Thanks for your interest!

So the main reason is the fact that the ResNet (except for the last layer) is pre-trained on ImageNet, as is standard practice for image retrieval evaluation on these datasets (the last layer is randomly initialised).

The primary reason why we decided to do this is because it is standard practice in the other works that we compared to. The explanation also makes sense that the last, randomly initialised layer requires a higher learning rate than the rest of the network that already has a good initialisation. 

We didn't do any ablation on this parameter, that would be interesting!

In this concurrent work at ECCV '20, I think they perform some studies on the learning rate on their FC "proxies" layer, as it is part of their contributions, and they show it to be important. https://arxiv.org/pdf/2004.01113.pdf
